examples/preact/tests/Count.js

```diff
@@ -0,0 +1,67 @@
+import { test } from 'uvu';
+import * as assert from 'uvu/assert';
+import * as ENV from './setup/env';
+
+// Must be require() –> @babel/register
+const Count = require('../src/Count.jsx').default;
```

Comment:
Hmm does it really needs to use commonjs because Babel/register? If we transpired modules with Babel as well could use standard esmodule here ?

Reply:
Ah, you're right! Although it only happens because `-r esm` still exists, which is turning everything from `import` -> `require` on the fly.

Updating the Svelte & Preact examples now